pubspec.yaml

```diff
@@ -8,9 +8,10 @@ author: Dart Team <misc@dartlang.org>
 environment:
   sdk: '>=1.20.0 <2.0.0'
 dependencies:
-  angular2: '>=2.2.0 <2.3.0'
+  angular2: 3.0.0-alpha
   intl: ^0.14.0
   js: ^0.6.1
+  meta: ^1.0.4
   observable: ^0.14.0+1
```

Comment:
Should make this `>=0.14.0+1 <0.21.0` – I think we just have deprecations, but nothing seems broken.